test/preview/pipeline/test_pipeline.py

```diff
@@ -28,29 +29,236 @@ def test_pipeline_store_api():
 
 
 @pytest.mark.unit
-def test_pipeline_stores_in_params():
+def test_pipeline_storemixin_one_existing_docstore():
```

Comment:
The names of the tests are not clear.
We're already in `test_pipeline.py` so it's clear we're testing that.
No test mentions the `add_component` method at all, that makes it harder to understand what we're testing.

For example I would change this to `test_add_component_using_existing_store`, `test_pipeline_storemixin_no_docstore` to `test_add_component_using_non_existing_store`.

I used those two as an example but other new tests should get the same treatment.